{
  "schema_version": "1.4.0",
  "id": "GHSA-4x83-5gw5-q346",
  "modified": "2022-05-02T03:17:24Z",
  "published": "2022-05-02T03:17:24Z",
  "aliases": [
    "CVE-2009-0668"
  ],
  "details": "Unspecified vulnerability in Zope Object Database (ZODB) before 3.8.2, when certain Zope Enterprise Objects (ZEO) database sharing is enabled, allows remote attackers to execute arbitrary Python code via vectors involving the ZEO network protocol.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2009-0668"
    },
    {
      "type": "WEB",
      "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/52377"
    },
    {
      "type": "WEB",
      "url": "http://mail.zope.org/pipermail/zope-announce/2009-August/002220.html"
    },
    {
      "type": "WEB",
      "url": "http://osvdb.org/56827"
    },
    {
      "type": "WEB",
      "url": "http://pypi.python.org/pypi/ZODB3/3.8.2#whats-new-in-zodb-3-8-2"
    },
    {
      "type": "WEB",
      "url": "http://secunia.com/advisories/36204"
    },
    {
      "type": "WEB",
      "url": "http://secunia.com/advisories/36205"
    },
    {
      "type": "WEB",
      "url": "http://www.securityfocus.com/bid/35987"
    },
    {
      "type": "WEB",
      "url": "http://www.vupen.com/english/advisories/2009/2217"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-94"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2009-08-07T19:30:00Z"
  }
}